elsafluss/rancid-tomatillos
# Feature display modal

#### What does this PR do?
As a user, when I click a movie, I am able to see that movies details. I am also able to close the window and reopen the grid of movies display.

#### Where should the reviewer start?
Modal component was constructed to accomplish the individual movie view.

#### How should this be manually tested?
run `npm start` and click a movie, the modal should open with the option to close the window.

#### Any background context you want to provide?
N/A
